recipes/qutip/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+${PYTHON} setup.py install
```

Comment:
Typically, anything that uses setuptools for installation should be a:

```
$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```

@ocefpaf - we need somewhere prominent to document that.